### PR TITLE
fix: skip cloud sync when endpoints are not deployed.

### PR DESCRIPTION
**Pull request by i4ki**

### CODE REVIEW

**Review by zied-elouaer — DISMISSED** [review]
LGTM

File: test/sandbox/sandbox.go [hunk 1]
```diff
@@ -607,7 +607,6 @@ func buildTree(t testing.TB, root *config.Root, layout []string) {
 		assert.NoError(t, err)
 
 		cfg.Stack = &hcl.Stack{}
-		cfg.Terramate = &hcl.Terramate{}
```
Comment (i4ki) [inline test/sandbox/sandbox.go:610]: minor and unrelated, but when I did `--log-level=warning` in the e2etests I saw the warning message of `terramate` block defined in non-root directory. The sandbox was still generating an empty terramate block when creating test stacks.

File: makefiles/common.mk [hunk 2]
```diff
@@ -100,6 +100,24 @@ release/dry-run:
...
+
+## sync the Terramate example stack with a failed status.
+.PHONY: cloud/sync/failed
+cloud/sync/failed: build test/helper
+	./bin/terramate --log-level=info		\
+			--disable-check-git-untracked   \
+			--disable-check-git-uncommitted \
+			run --cloud-sync-deployment --  \
+			$(PWD)/bin/helper false
+
```
Comment (i4ki) [inline makefiles/common.mk:120]: just helper rules to speed up manual testing.

**Review by MordecaiMalignatus — APPROVED** [review]
I'm going to steal some of your test fixture patterns, they're elegant.

**Pull request merged**